packages/hw-app-avalanche/src/Avalanche.js

```diff
@@ -137,10 +137,16 @@ export default class Avalanche {
     let lastData = rawTx.slice(lastOffset, lastOffset+this.MAX_APDU_SIZE);
     let response = await this.transport.send(0x80, this.INS_SIGN_HASH, 0x81, 0x00, lastData);
     let resp = Buffer.from(response,"hex");
```

Comment:
Why does this line even exist, again?  I mean, I thought `transport.send` returned, like, a binary buffer?   But we hex-decode it here?

Can we just get rid of this line altogether?

Reply:
Yeah I think you're right.